FAQ/FAQ-Integrations.md

```diff
@@ -26,7 +26,7 @@ At this time, BTCPay does not have a Shopify integration.
 Yes, you can. While various e-commerce CMS use integrations,  you can use BTCPay even if you're not a merchant. For more information about use-cases, see [this page](/UseCase.md)
 ## WooCommerce FAQ
 ### How to configure order status in WooCommerce?
-Order status depends on a merchant's business model. To better understand BTCPay order(invoice) status [read this document](/WooCommerce.md#btcpay-order-statuses).
-There's no generally the best way to configure them without trial and error and seeing what works for your business. See this discussion for [more information on order statuses](https://nbitstack.com/t/how-to-set-up-order-statuses-in-woocommerce-and-btcpay/67).
+Order status depends on a merchant's business model. To better understand BTCPay order (invoice) status [read this document](/WooCommerce.md#btcpay-order-statuses).
+There's no best way to configure them without trial and error and seeing what works for your business.
 ### How to customize e-mail confirmations in WooCommerce?
-If you wish to send an e-mail after a certain status to the customer, you can configure your BTCPay WooCommerce plugin. This is recommended only if you know what you're doing. [Follow this community guide](https://nbitstack.com/t/how-to-customize-transaction-e-mail-confirmations-in-btcpay-and-woocommerce/75).
+If you wish to send an e-mail after a certain status to the customer, you can configure your BTCPay WooCommerce plugin. This is recommended only if you know what you're doing. 
```

Comment:
This is now confusing a bit without that link, you cannot configure that in BTCPay WooCommerce. 
You need to edit WooCommerce order e-mail templates.
Maybe link to this guide https://www.cloudways.com/blog/how-to-customize-woocommerce-order-emails/